websites/T/TuneIn/presence.ts

```diff
@@ -19,36 +19,34 @@ function getTimestamps(
   audioTime: string,
   audioDuration: string
 ): Array<number> {
-  var splitAudioTime = audioTime.split(":").reverse();
-  var splitAudioDuration = audioDuration.split(":").reverse();
+  const splitAudioTime = audioTime.split(":").reverse(),
+   splitAudioDuration = audioDuration.split(":").reverse(),
 
-  var parsedAudioTime = getTime(splitAudioTime);
-  var parsedAudioDuration = getTime(splitAudioDuration);
+   parsedAudioTime = getTime(splitAudioTime),
+   parsedAudioDuration = getTime(splitAudioDuration),
 
-  var startTime = Date.now();
-  var endTime =
+  startTime = Date.now(),
+   endTime =
     Math.floor(startTime / 1000) - parsedAudioTime + parsedAudioDuration;
   return [Math.floor(startTime / 1000), endTime];
 }
 
-var elapsed = Math.floor(Date.now() / 1000);
-var title, author;
+let elapsed = Math.floor(Date.now() / 1000), title, author;
```

Comment:
i think elapsed can be const

Reply:
![image](https://user-images.githubusercontent.com/42322979/118945687-f9d86800-b955-11eb-9d1b-e042e4c06b4b.png)

Reply:
it can't be null since it's assigned right away